content/rc/accounts/account-settings.md

```diff
@@ -0,0 +1,75 @@
+
+- The **Cloud Accounts** tab is displayed for Flexible (or Annual) subscriptions hosted on Amazon Web Services (AWS).  To learn more, see [Manage AWS cloud accounts]({{<relref "rc/cloud-integrations/aws-cloud-accounts/">}})  
+    
+## Account info settings
+
+The **Account Info** section provides basic details about your account, including:
+
+| Setting | Description |
+|---------|-------------|
+| _Owner name_ | Person associated with the Redis Cloud account |
+| _Account name_ | Organization associated with the Redis Cloud account | 
+| _Date created_ | Date the user's Redis Cloud account was created, which may differ from the organization account creation date |
+| <nobr>_Owner email address_</nobr> | Email address used to create the owner's account |
+| _Account number_ | Internal ID of the owner's account |
+| _Last updated_ | Date of the last administrative change to the owner's account, typically reflect access changes or other administrative updates | 
+
+You cannot change the email address associated with a Redis Cloud account.  Instead, create a new account with the updated email address, assign it as an administrator to the organization account, and then use the new account to delete the account with the invalid email address.
+
+## Account address settings
+
+The **Account address** section shows the address associate with the current Redis Cloud account and the current time zone. 
+
+To update the time zone, select the desired time zone from the **Time zone** drop-down.
+
+In addition, this section may include fields unique to your location.  For example, certain regions require tax IDs or other regulatory details.
+
+When updating details in this section, pay particular attention to prompts and error messages.
+
+## Security settings
+
+The security section lets you:
+
+The security section lets you:
+
+- Manage [multi-factor authentication]({{<relref "/rc/security/multi-factor-authentication">}}) (MFA) for your Redis Cloud account 
+
+- Download [certificate authority]({{<relref "/rc/security/database-security/tls-ssl#certificates">}}) (CA) certificates for the certificates associated with your Redis Cloud account
+
+## Integration settings
+
+The **Integrations settings** section includes settings that help you manage the integration of your Redis Cloud account with your underlying cloud provider.  For example, subscriptions hosted on AWS can [map their accounts]({{<relref "/rc/administration/aws-zone-mapping">}}) to specific availability zones.
+
+If this section doesn't appear on the **Account Settings** screen, it generally means that there aren't any settings for current account to manage.
+
+## Save or discard changes
+
+Few account settings can be changed, however, you can update a few details, such as **Time Zone**, **MFA enforcement**, and AWS availability zone mapping.
```

Comment:
Maybe reword this slightly, break up the sentence, or change the comma before "however" to a semicolon?
```suggestion
Although most account settings cannot be changed, you can update a few details, such as **Time Zone**, **MFA enforcement**, and AWS availability zone mapping.
```

Reply:
Fixed.